android/assets/jsons/translations/template.properties

```diff
@@ -228,6 +228,14 @@ Please don't spread your religion to us. =
 Don't spread religion in our cities. = 
 Very well, we shall spread our faith elsewhere. = 
 We noticed you have continued spreading your faith, despite your promise. This will have...consequences. = 
+
+# Will be deprecated in a few versions
+Please don't spy on us. = 
+Please don't spy on us. = 
+Very well, we shall no longer spy on you. = 
```

Comment:
Too direct. Something like "We see our people are not welcome in your lands... we will take our attention elsewhere"